recipes/atlas-cli-bulk/meta.yaml

```diff
@@ -10,15 +10,15 @@ source:
 
 build:
   number: 0
-  skip: true # [win32]
+  skip: false # [win32]
 
 requirements:
   build:
   host:
   run:
     - bash
-    - openjdk
-    - gradle
+    - openjdk >=11.0.0.0
+    - gradle >= 7.0.0
```

Comment:
Set Gradle 7 version explicitly, since we’re using some specific features that aren’t backwards compatible with Gradle 8.